.gitignore

```diff
@@ -29,6 +29,7 @@ bin/node-template/Cargo.lock
 nohup.out
 polkadot_argument_parsing
 polkadot.*
+!docs/sdk/src/polkadot_sdk/polkadot.rs
```

Comment:
what's this?

Reply:
line 31 was excluding that file from git, so I added that line to not exclude it